android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -871,6 +873,14 @@ public void testFieldsExist() throws Exception {
         Assert.assertTrue(
                 fieldExists("org/chromium/chrome/browser/sync/settings/ManageSyncSettings",
                         "mSyncPaymentsIntegration"));
+        Assert.assertTrue(fieldExists(
+                "org/chromium/chrome/browser/sync/settings/ManageSyncSettings", "mSyncEverything"));
```

Comment:
We will likely need test proguard rules for tests to run in release

Reply:
thanks, I missed that, run test on Debug and didn't see error.
Fixed at https://github.com/brave/brave-core/pull/19477/commits/9b1155a55327056ee8ad214fa64abda97f438455